src/main/java/vowelExtract/stringVowel.java

```diff
@@ -0,0 +1,25 @@
+package vowelExtract;
+import java.util.Scanner;  // Import statement for Scanner
+
+public class stringVowel {
+
```

Comment:
Java convention is to capitalize class names `StringVowel `

Reply:
ok I will change this